src/content/chainlink-functions/tutorials/automate-functions-custom-logic.mdx

```diff
@@ -47,36 +47,36 @@ You can locate the scripts used in this tutorial in the [_examples/10-automate-f
    npx env-enc set
    ```
 
-### Deploy an Automated Functions Consumer contract
+### Deploy a Custom Automated Functions Consumer contract
```

Comment:
```suggestion
### Deploy a Functions Consumer contract
```

Let's align all the titles (and keep them concise). Too wordy (confusing) otherwise